### PR TITLE
Display default core language in "wp core version --extra".

### LINKED ISSUE #3219: Display default core language
*Please keep this 'good-first-issue' for a first time contributor if possible. Thanks!*

Could we add `$wp_local_package` to `core version --extra`?
If it is not available then display `en_US`.

### PR DESCRIPTION
**Pull request by rahulsprajapati**
Fixes #3219

### CODE REVIEW

File: features/core-version.feature [hunk 1]
```diff
@@ -16,4 +16,5 @@ Feature: Find version for WordPress install
       WordPress version: 4.4.2
       Database revision: 35700
       TinyMCE version:   4.208 (4208-20151113)
+      Language:          en_US
```
Comment (szepeviktor) [inline features/core-version.feature]: Please write `Package Language` here also.

File: php/commands/core.php [hunk 2]
```diff
@@ -945,9 +945,10 @@ public function version( $args = array(), $assoc_args = array() ) {
 			}
 
 			echo \WP_CLI\Utils\mustache_render( 'versions.mustache', array(
-				'wp-version'  => $details['wp_version'],
-				'db-version'  => $details['wp_db_version'],
-				'mce-version' => ( $human_readable_tiny_mce ?
+				'wp-version'    => $details[ 'wp_version' ],
+				'db-version'    => $details[ 'wp_db_version' ],
+				'local-package' => ! empty( $details[ 'wp_local_package' ] ) ? $details[ 'wp_local_package' ] : 'en_US',
```
Comment (szepeviktor) [inline php/commands/core.php]: You couls spare a `!` by reversing the two.

File: php/commands/core.php [hunk 3]
```diff
@@ -945,9 +945,10 @@ public function version( $args = array(), $assoc_args = array() ) {
 			}
 
 			echo \WP_CLI\Utils\mustache_render( 'versions.mustache', array(
-				'wp-version'  => $details['wp_version'],
-				'db-version'  => $details['wp_db_version'],
-				'mce-version' => ( $human_readable_tiny_mce ?
+				'wp-version'    => $details[ 'wp_version' ],
```
Comment (szepeviktor) [inline php/commands/core.php]: Actually WPCS say `$arrray['contant-without-surrounding-spaces']`

File: php/commands/core.php (hunk 2 above)
Comment (rahulsprajapati) [inline php/commands/core.php]: ok

**Comment by danielbachhuber** [comment]
@rahulsprajapati Why'd you close this pull request?

**Comment by rahulsprajapati** [comment]
@danielbachhuber oops, I was thinking that my PR was closed.

**Comment by rahulsprajapati** [comment]
@danielbachhuber Please can you suggest me what should I use for language : **Package language** OR **Installation language**.

File: features/core-version.feature [hunk 4]
```diff
@@ -16,4 +16,5 @@ Feature: Find version for WordPress install
       WordPress version: 4.4.2
       Database revision: 35700
       TinyMCE version:   4.208 (4208-20151113)
+      Package Language:  en_US
```
Comment (danielbachhuber) [inline features/core-version.feature]: Can we call this "Package language" (note the lowercase "language") for consistency with the other keys?

**Comment by danielbachhuber** [comment]
> Please can you suggest me what should I use for language : Package language OR Installation language.

The former is fine.

Can you include a test case for a non-en_US locale too?

File: features/core-version.feature (hunk 4 above)
Comment (rahulsprajapati) [inline features/core-version.feature]: yes. :+1:

**Comment by rahulsprajapati** [comment]
@danielbachhuber Yes, I will include asap.

File: features/core-version.feature [hunk 5]
```diff
@@ -16,4 +16,24 @@ Feature: Find version for WordPress install
...
+      Success: WordPress downloaded.
+      """
+
+    When I run `wp core version --extra`
+    Then STDOUT should be:
+      """
+      WordPress version: 4.4.2
+      Database revision: 35700
+      TinyMCE version:   4.208 (4208-20151113)
+      Package language:  de_DE
```
Comment (danielbachhuber) [inline features/core-version.feature]: Nice!

Comment (rahulsprajapati) [inline features/core-version.feature]: Thank you. :)

**Pull request merged**